src/Client/Core/DurableTaskClient.cs

```diff
@@ -440,6 +440,25 @@ public virtual Task<string> RestartAsync(
         CancellationToken cancellation = default)
         => throw new NotSupportedException($"{this.GetType()} does not support orchestration restart.");
 
+    /// <summary>
+    /// Rewinds the specified orchestration instance by re-executing any failed Activities, and recursively rewinding
+    /// any failed suborchestrations with failed Activities.
+    /// The orchestration's history will be replaced with a new history that excludes the failed Activities and suborchestrations,

```

Comment:
nit: The `<summary>...</summary` should just be a once sentence description of the API. The other details around behavior should be in the `<remarks>...</remarks>` section.